cmd/datablue/handlers.go

```diff
@@ -538,7 +538,5 @@ func writeDeviceError(w http.ResponseWriter, dev *iotds.Device, err error) {
 	}
 	w.Header().Add("Content-Type", "application/json")
 	fmt.Fprint(w, `{"er":"`+err.Error()+`"`+rc+`}`)
-	if debug {
-		log.Println("Wrote error: " + err.Error())
-	}
+	log.Println("Wrote error: " + err.Error())
```

Comment:
Now that's we're always logging, let's make it slightly more descriptive, e.g., "Wrote device error".